### ISSUE TITLE
Additional multiline option

### ISSUE THREAD

**Issue by joaomcteixeira** [timothycrosley/isort#1093]
Multi line option 3 allows for:

```
from third_party import (
    lib1,
    lib2,
    lib3,
    lib4,
)
```

Though with flake8 we can set up the `hang-closing = true` option with allows for:

```
from third_party import (
    lib1,
    lib2,
    lib3,
    lib4,
    )  <- mind here the 4 spaces
```

Having both options activated conflicts either with one or another. Is it possible to add to `isort` another multiline option that would allow for the `)` to be indented?

Thanks.

**Comment by timothycrosley** [comment]
While I don't have the bandwith to add support for this requested feature myself, I will accept a pull request that adds it. The correct place to add this functionality is here: https://github.com/timothycrosley/isort/blob/develop/isort/wrap_modes.py#L46

**Comment by joaomcteixeira** [comment]
Thanks for you encouragement, please see PR #1097.

**Comment by joaomcteixeira** [comment]
Closed by #1097
